BlueprintUI/Sources/Measuring/SizeConstraint.swift

```diff
@@ -176,3 +176,33 @@ extension SizeConstraint {
 
     }
 }
+
+extension SizeConstraint {
+    /// This property wrapper checks the value of `atMost` cases, and turns it into an
+    /// `unconstrained` axis if the value equals `greatestFiniteMagnitude` or `isInfinite`.
+    @propertyWrapper public struct UnconstrainedInfiteAxis: Equatable, Hashable {
+        private var correctedAxis: Axis
+
+        public var wrappedValue: Axis {
+            get { correctedAxis }
+            set { correctedAxis = Self.correctedAxis(for: newValue) }
+        }
+
+        public init(wrappedValue value: Axis) {
+            correctedAxis = Self.correctedAxis(for: value)
+        }
+
+        private static func correctedAxis(for axis: Axis) -> Axis {
```

Comment:
Mind adding a unit test for this behavior?

Reply:
Done!